wasm/wazeroir/operations.go

```diff
@@ -338,11 +338,15 @@ const (
 )
 
 type Label struct {
-	FrameID uint32
-	Kind    LabelKind
+	FrameID          uint32
+	OriginalStackLen int
+	Kind             LabelKind
 }
 
 func (l *Label) String() (ret string) {
+	if l == nil {
```

Comment:
comment why handling nil unless all things are doing that, then never mind

Reply:
https://github.com/tetratelabs/wazero/pull/60/commits/5fd13b36250f0fb937bfc3867b52f4848d7f0022